### PR TITLE
[state] A few nits after consolidation of state.

### PR DESCRIPTION
**Pull request by ejgallego**
We remove some dead aliases and add some documentation to the
interface file.

### CODE REVIEW

File: vernac/vernacstate.mli
Comment (gares) [inline vernac/vernacstate.mli:76]: I don't see why you are renaming this. `pstate` and `non_pstate` seems more informative to me than `pstate` and `rstate`.

**Review by SkySkimmer — COMMENTED** [review]
I'm going to leave this to @coq/stm-maintainers

File: vernac/vernacstate.mli
Comment (ejgallego) [reply vernac/vernacstate.mli:76]: Was meant to be "rest of state", I've kept `non_pstate` .

**Review by gares — APPROVED** [review]

**Comment by gares** [comment]
@ejgallego would you mind rebasing? (CI run is quite old)

**Comment by ejgallego** [comment]
> @ejgallego would you mind rebasing? (CI run is quite old)

Done

**Comment by gares** [comment]
@coqbot: merge now

**Pull request merged**
